Rakefile

```diff
@@ -13,3 +13,28 @@ begin
   load 'jasmine/tasks/jasmine.rake'
 rescue LoadError
 end
+
+# Clear noisy and unusable tasks added by rspec-rails
+if defined?(RSpec)
+  task("spec").clear
+  %w(
+    automation
+    controllers
+    helpers
+    initializers
+    lib
+    mailers
+    migrations
+    models
+    presenters
+    replication
+    requests
+    routing
+    services
+    task_helpers
+    tools
+    views
+  ).each do |type|
```

Comment:
Not that I can think of :( Ideally I'd like to see a way to keep railties from loading their tasks; If anyone has an idea of how to do this, that'd be a much better solution than clearing them like this.